lib/helpers/style.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:flutter/material.dart';
+
+Color light = const Color(0xFFF7F8FC);
+Color lightGrey = const Color(0xFFA4A6B3);
+Color dark = const Color(0xFF363740);
+Color active = const Color(0xFF3C19C0);
+Color lightBlue = const Color(0xFF01579B);
```

Comment:
I would love to see this color match with the blue on the logo.

Reply:
the changes you requested has been fixed

Reply:
Yes, I notice the change. However, based on the image shared on WhatsApp, can we now have the colors for the button and the text changed to the orange? Ultimately, we should be having an additional color of orange to the palette.

Then you should also be adding those buttons for social logins as in the shared image.
Also consider putting the same exact wordings as in the shared image.